app/controllers/users_controller.rb

```diff
@@ -9,18 +12,54 @@ def new
     @user = User.new
   end
 
+  # POST /users
   def create
     @user = User.new user_params
 
     if @user.save
-      log_in @user
+      @user.send_activation_email
+      flash[:info] = t ".check_email"
+      redirect_to root_url
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  # GET /users/:id/edit
+  def edit
+    return if @user
+
+    flash[:danger] = t ".not_found"
+    redirect_to root_path
+  end
```

Comment:
```suggestion
  def edit; end
```
Thừa, có hàm load_user r